recipe/meta.yaml

```diff
@@ -221,7 +227,7 @@ about:
     For best performance, the maintainers of the package
     [recommend](https://github.com/conda-forge/staged-recipes/pull/11337#issuecomment-623718460)
     using the MKL implementation of blas/lapack. You can ensure that this is installed
-    by adding "libblas =*=mkl" to your dependencies.
+    by adding "libblas =*=*mkl" to your dependencies.
```

Comment:
Should we add this to a [`post-link` script]( https://docs.conda.io/projects/conda-build/en/latest/resources/link-scripts.html ) for greater visibility?

Reply:
OK, will look into this.